requirements.txt

```diff
@@ -0,0 +1 @@
+ipython
```

Comment:
To configure the environment in development mode, But I believe the file name should be 
requirements_dev.txt. What do you think ?